installer/config.yaml

```diff
@@ -27,10 +27,12 @@ tssc:
         manageSubscription: true
         # namespacePrefixes:
         #   - tssc-app
-        # RBAC:
+        authProvider: github
+        # Possible values: github, gitlab, microsoft
+        RBAC:
```

Comment:
This line should be commented out as well, no?

Reply:
yes, good catch